core/src/main/kotlin/com/strumenta/kolasu/model/Node.kt

```diff
@@ -229,4 +225,8 @@ open class Node() : Serializable {
         val ref: ReferenceByName<T> = getReference(referenceName)
         ref.referred = referred
     }
+
+    fun subscribe(observer: Observer<NodeNotification<in Node>>) {
+        this.changes.subscribe(observer)
```

Comment:
Same here

Reply:
<img width="1343" alt="Screenshot 2023-06-19 at 17 14 03" src="https://github.com/Strumenta/kolasu/assets/439078/f3713b03-43f7-4a45-82db-a9597595cccf">

Reply:
Oh OK I was only aware of `subscribe` overloads returning `Disposable`